test/jdk/java/nio/file/attribute/BasicFileAttributeView/SetTimesNanos.java

```diff
@@ -24,7 +24,8 @@
 /* @test
  * @bug 8181493 8231174 8343417
  * @summary Verify that nanosecond precision is maintained for file timestamps
- * @requires (os.family == "linux") | (os.family == "mac") | (os.family == "windows")
+ * @requires (os.family == "aix") | (os.family == "linux") |
+ *           (os.family == "mac") | (os.family == "windows")
  * @library ../.. /test/lib
```

Comment:
I'll run another set of testing. Should the test work on AIX, I think we can remove the `@requires` section altogether.

Reply:
Yes, I think `@requires` can be dropped now.

Reply:
Yes, the test has passed on AIX.

Reply:
I will delete the `@requires` tag.

Reply:
`@requires` deleted in fc5f8e8.